satpy/readers/netcdf_utils.py

```diff
@@ -217,10 +228,7 @@ def _get_var_from_xr(self, group, key):
     def _get_var_from_filehandle(self, group, key):
         # Not getting coordinates as this is more work, therefore more
         # overhead, and those are not used downstream.
-        if group is None:
-            g = self.file_handle
-        else:
-            g = self.file_handle[group]
+        g = self.file_handle[group]
```

Comment:
From what I see `group` can still be None: https://github.com/pytroll/satpy/pull/1219/files#diff-cfed382dd19ba8d52b7734cecf817407R197-R200
Is that an accepted key ?